linux/ibus-kmfl/src/kmflutil.c

```diff
@@ -308,11 +308,7 @@ ibus_kmfl_list_engines (void)
     engines = ibus_kmfl_add_engines(engines, keyboard_list);
     g_list_free(keyboard_list);
 
-    keyboard_list = kmfl_get_keyboard_list("/usr/share/keyman");
```

Comment:
> 
> 
> Since this ibus-kmfl has only started using this directory in the current alpha, this shouldn't affect regular users once it stops looking for keyboards in `/usr/share/keyman`. The ability to load keyboards from here will require the installation of the new ibus-keyman sister package when complete.

Documented